fibo.py

```diff
@@ -0,0 +1,5 @@
+def fibonacci_ruim(n):
```

Comment:
Consider using a more descriptive function name than `fibonacci_ruim` to enhance code readability.
```suggestion
def fibonacci(n):
```